integration/test/chart/main_test.go

```diff
@@ -0,0 +1,81 @@
+// +build k8srequired
+
+package chart
+
+import (
+	"context"
+	"testing"
+
+	"github.com/giantswarm/e2e-harness/pkg/framework"
+	"github.com/giantswarm/helmclient"
+	"github.com/giantswarm/micrologger"
+	"github.com/giantswarm/operatorkit/client/k8scrdclient"
+
+	"github.com/giantswarm/app-operator/integration/setup"
+)
+
+var (
+	h          *framework.Host
+	helmClient *helmclient.Client
+	l          micrologger.Logger
+	crdClient  *k8scrdclient.CRDClient
+)
+
+func init() {
```

Comment:
There are a few clients missing in your setup here, this piece of code should give you all the necessary clients:
https://github.com/giantswarm/aws-operator/blob/master/integration/setup/config.go#L49-L134